packages/fmg-core/tsconfig.json

```diff
@@ -2,12 +2,13 @@
   "compilerOptions": {
     /* Basic Options */
     "lib": ["es5", "es6"],
-    "allowJs": true,                        /* Allow javascript files to be compiled. */
```

Comment:
There are still a bunch of `js` files that haven't been rewritten yet.

Reply:
I couldn't see any ... ?

Reply:
... you're right